software-engineering-policies/InfrastructureAsCode/terraform.md

```diff
@@ -0,0 +1,38 @@
+# Infrastructure as Code: Terraform
+
+Terraform is our tool of choice, and should be considered in the first instance.
+
+Most standards and practises at the UKHO is covered in the Training courses we provide around the use of Terraform.
+
+## Avoid adding local modules
```

Comment:
In this section could you spell out where you want them to go. I.E Modules should be stored in a github repo created for them.

Reply:
Think I have done that now?